firebase-firestore/src/main/java/com/google/firebase/firestore/remote/Datastore.java

```diff
@@ -246,8 +247,13 @@ public Task<Map<String, Value>> runAggregateQuery(
                 throw task.getException();
               }
 
+              Map<String, Value> result = new HashMap<>();
```

Comment:
Consider porting [the comment from the reference PR](https://github.com/firebase/firebase-js-sdk/pull/7244/files#diff-adbaf74dbe898e542aac999d5463558c85dba97d37c32124dc96c1a3db243c1dR280)